framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -456,6 +456,17 @@ int cfg_notify_iface_status(const BPL_INTERFACE_STATUS_NOTIFICATION *status_noti
  */
 int cfg_get_administrator_credentials(char pass[BPL_USER_PASS_LEN]);
 
+/**
+ * Returns the STA interface for the specified radio id.
+ *
+ * @param [in] iface Interface name for the requested parameters.
+ * @param [out] sta_iface name of STA interface (up to 32 bytes in length).
+ *
+ * @return 0 Success.
+ * @return -1 Error.
+ */
+int cfg_get_sta_iface(const char iface[BPL_IFNAME_LEN], char sta_iface[BPL_IFNAME_LEN]);
```

Comment:
I think this should have been using explicit size for strings:
```
int cfg_get_sta_iface(const char *iface, size_t iface_length, char *sta_iface, size_t sta_iface_length);
```

Since this is not the case in the whole BPL UCI implementation - (nitpick)

Reply:
skipped for now